examples/javaee/README.md

```diff
@@ -0,0 +1,158 @@
+
+Here is the config for MySQL pod: [mysql-pod.yaml](mysql-pod.yaml)
+
+Create the MySQL pod as follows:
+
+```sh
+kubectl create -f examples/javaee/mysql-pod.yaml
+```
+
+Check status of the pod as:
+
+```sh
+kubectl.sh get -w po
+NAME        READY     STATUS    RESTARTS   AGE
+mysql-pod   0/1       Pending   0          4s
+NAME        READY     STATUS    RESTARTS   AGE
+mysql-pod   0/1       Running   0          44s
+mysql-pod   1/1       Running   0         44s
+```
+
+Wait for the status to `1/1` and `Running`.
+
+### Start MySQL Service
+
+In Kubernetes a [_Service_](../../docs/user-guide/services.md) describes a set of Pods that perform the same task. An important use for a Service is to create a load balancer which distributes traffic across members of the set.  But a Service can also be used as a standing query which makes a dynamically changing set of Pods (or the single Pod we've already created) available via the Kubernetes API.
+
+The order of Service and the targeted Pods does not matter. However Service needs to be started before any other Pods consuming the Service are started.
+
+In this application, we will use a Kubernetes Service to provide a discoverable endpoints for the MySQL endpoint in the cluster.  MySQL service target pods with the labels `name: mysql-pod` and `context: docker-k8s-lab`.
+
+Here is definition of the MySQL service: [mysql-service.yaml](mysql-service.yaml)
+
+Create this service:
+
+```sh
+kubectl create -f examples/javaee/mysql-service.yaml
+```
+
+Get status of the service:
+
+```sh
+kubectl.sh get -w se
+NAME            LABELS                                    SELECTOR                                IP(S)          PORT(S)
+kubernetes      component=apiserver,provider=kubernetes   <none>                                  10.247.0.1     443/TCP
+mysql-service   context=docker-k8s-lab,name=mysql-pod     context=docker-k8s-lab,name=mysql-pod   10.247.63.43   3306/TCP
+```
+
+If multiple services are running, then it can be narrowed by specifying labels:
+
+```sh
+kubectl.sh  get -w po -l context=docker-k8s-lab,name=mysql-pod
+NAME        READY     STATUS    RESTARTS   AGE
+mysql-pod   1/1       Running   0          4m
+```
+
+This is also the selector label used by service to target pods.
+
+When a Service is run on a node, the kubelet adds a set of environment variables for each active Service. It supports both Docker links compatible variables and simpler `{SVCNAME}_SERVICE_HOST` and `{SVCNAME}_SERVICE_PORT` variables, where the Service name is upper-cased and dashes are converted to underscores.
+
+Our service name is ``mysql-service'' and so MYSQL_SERVICE_SERVICE_HOST and MYSQL_SERVICE_SERVICE_PORT variables are available to other pods. This host and port variables are then used to create the JDBC resource in WildFly.
```

Comment:
ticks around mysql aren't formatting correctly in markdown. Pleas put env variables in ticks as well.